solidity-v1/contracts/PhasedEscrow.sol

```diff
@@ -183,6 +183,30 @@ interface IStakingPoolRewards {
     function notifyRewardAmount(uint256 amount) external;
 }
 
+/// @title PlainTransferEscrowBeneficiary
+/// @notice  PlainTransferEscrowBeneficiary is a beneficiary contract that can
+///          receive a withdrawal order from PhasedEscrow and BatchedPhasedEscrow
+///          contracts. It immediately sends the received tokens to the recipient
+///          set by the beneficiary deployer.
+///          The recipient does not have to implement any custom functions, it can
+///          be a smart contract or EOA - beneficiary is executing a plain ERC20
+///          transfer.
+contract PlainTransferEscrowBeneficiary is Ownable, IBeneficiaryContract {
```

Comment:
There is no point in changing that because it's probably the last time we deploy a beneficiary. And the last time we'll use any beneficiary will be in April.

Anyway, I think keeping them together makes sense as they are very closely related.